tests/test_models/test_decode_heads/test_cylinder3d_head.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from unittest import TestCase
+
+import torch
+from mmcv.ops import SparseConvTensor
+
+from mmdet3d.models.decode_heads import Cylinder3DHead
+from mmdet3d.structures import Det3DDataSample, PointData
+
+
+class TestCylinder3DHead(TestCase):
+
+    def test_cylinder3d_head_loss(self):
+        """Tests DGCNN head loss."""
+
```

Comment:
if not torch.cuda.is_available():
            pytest.skip('test requires GPU and torch+cuda')